test/jobs/github-event/push.js

```diff
@@ -379,16 +392,63 @@ test('github-event push', async t => {
     t.notOk(repo.enabled)
     t.end()
   })
+
+  t.test('package.json deleted on private repo', async t => {
```

Comment:
❓ why did you add this test?

Reply:
Same as https://github.com/greenkeeperio/greenkeeper/pull/602#discussion_r164702375 :D